react-native/training/src/app/screens/TodoList/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import actionCreators from '../../../redux/todoList/actions';
+import { itemsTodoList } from '../../../propTypes/propTypes';
+
+import TodoList from './layout';
+
+function TodoListContainer({ items, addItem, removeItem, toggleItemCompleted, removeCompleted }) {
+  return (
+    <TodoList
+      items={items}
+      addItem={addItem}
+      removeItem={removeItem}
+      toggleItemCompleted={toggleItemCompleted}
+      removeCompleted={removeCompleted}
+    />
+  );
+}
+
+TodoListContainer.propTypes = {
+  items: itemsTodoList,
+  addItem: PropTypes.func.isRequired,
+  removeItem: PropTypes.func.isRequired,
+  toggleItemCompleted: PropTypes.func.isRequired,
+  removeCompleted: PropTypes.func.isRequired
+};
+
+const mapStateToProps = ({ todoList }) => ({
```

Comment:
Try to not apply destructuring in `mapStateToProps` and `mapDispatchToProps`. Goes against to snippets